backend/src/daemon/fwupdmgr.rs

```diff
@@ -83,32 +31,16 @@ impl From<std::io::Error> for Error {
 
 impl From<std::str::Utf8Error> for Error {
     fn from(_e: std::str::Utf8Error) -> Self {
-        Error::Json
-    }
-}
-
-impl From<serde_json::Error> for Error {
-    fn from(_e: serde_json::Error) -> Self {
-        Error::Json
-    }
-}
-
-impl From<std::num::ParseIntError> for Error {
-    fn from(_e: std::num::ParseIntError) -> Self {
-        Error::Json
+        Error::Utf8
     }
 }
 
 pub fn is_launch_updated() -> Result<bool, Error> {
-    // Get all fwupd devices
     let stdout = Command::new("fwupdmgr")
-        .args(["get-devices", "--json"])
+        .args(["get-updates", "--json"])
         .output()?
         .stdout;
-    info!("before json");
-    let json = std::str::from_utf8(&stdout)?;
-    info!("before devices");
-    let devices: Devices = serde_json::from_str(json).unwrap();
+    let stdout = std::str::from_utf8(&stdout)?;
 
-    devices.get_launch()?.is_newer_release()
+    Ok(!stdout.contains("Configurable Keyboard"))
```

Comment:
We already depend on the `regex` crate so we could easily use a regex match here for `Launch.* Configurable Keyboard`. Though yeah, it probably doesn't make a difference in practice.

Reply:
Easy enough [eb664f0](https://github.com/pop-os/keyboard-configurator/pull/156/commits/eb664f0d95f9fcb813f6bfbc7191bd595d7bef34)